plugins/spotify/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/spotify/logic/PlayerLogic.java

```diff
@@ -321,13 +361,21 @@ void setSuppressed(boolean suppressed) {
 				if (this.postSuppressionAction == PostSuppressionAction.PAUSE) {
 					// Already paused, do nothing
 					this.postSuppressionAction = PostSuppressionAction.NONE;
+				} else if (this.postSuppressionAction == PostSuppressionAction.PLAY_TRACK) {
+					this.spotifyAPICalls.playSongFromUri(this.trackUriToPlay);
+					this.postSuppressionAction = PostSuppressionAction.NONE;
+				} else if (this.postSuppressionAction == PostSuppressionAction.PLAY_COLLECTION) {
+					this.spotifyAPICalls.playListFromUri(this.collectionUriToPlay);
+					this.postSuppressionAction = PostSuppressionAction.NONE;
 				} else {
 					resume();
 				}
 				this.suppressed = false;
-			} else if (suppressed && isPlaying) {
+			} else if (suppressed) {
+				if(isPlaying) {
+					pause();
+				}
 				// Consider pausing playback
-				pause();
 				this.suppressed = true;
```

Comment:
this is potentially dangerous. When I initially designed the suppression-System, I assumed that `suppressed` is only ever true when playback was actually interrupted.